action.yml

```diff
@@ -31,4 +38,5 @@ outputs:
     description: "The Success/Failure of the action"
 runs:
   using: 'docker'
-  image: 'Dockerfile'
+  # image: 'Dockerfile'
+  image: docker://quay.io/aptible/aptible-deploy-action:v2
```

Comment:
Are you re-using this tag from last time? And did you build and push properly?

Reply:
Yes, same tag.  I just built and pushed our recent changes.